src/conda_package_handling/conda_fmt.py

```diff
@@ -134,6 +139,10 @@ def tell(self):
                     component_tar.close()
                     component_stream.close()
 
+        with open(conda_pkg_fn, "rb") as f:
+            sha256 = hashlib.sha256(f.read()).hexdigest()
+            LOG.info("Created '%s' with sha256 '%s'", conda_pkg_fn, sha256)
```

Comment:
We might want to do this in chunks. Some `conda` packages might be too large to be read in memory all at once.

Reply:
added chunking